Content.Shared/Blocking/Components/SharedBlockingComponent.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Sound;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Blocking;
+
+/// <summary>
+/// This component goes on an item that you want to use to block
+/// </summary>
+[RegisterComponent]
+public sealed class SharedBlockingComponent : Component
+{
+    /// <summary>
+    /// The entity that's blocking
+    /// </summary>
+    [ViewVariables]
+    public EntityUid? User;
+
+    /// <summary>
+    /// Is it currently blocking?
+    /// </summary>
+    [ViewVariables]
+    public bool IsBlocking;
+
+    /// <summary>
+    /// The ID for the fixture that's dynamically created when blocking
+    /// </summary>
+    public string BlockFixtureID = "blocking-active";
+
+    /// <summary>
+    /// The shape of the blocking fixture that will be dynamically spawned
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("blockRadius")]
+    public float BlockRadius = 0.5f;
```

Comment:
Is this supposed to be a shape?

Reply:
Yep a nice big circle

Reply:
Don't you need radius for a circle shape, though?

Reply:
I didn't realize there could just be a separate shape field from a fixture component.